Dockerfile

```diff
@@ -10,6 +10,7 @@ LABEL org.label-schema.schema-version="1.0" \
     org.label-schema.vendor="Alfresco"
 
 RUN yum -y update \
+    python-2.7.5-86.el7 \
```

Comment:
Can't you update to the latest?  E.g. 0:2.7.5-88.el7

Reply:
86 is latest available sadly, as for all the others .

thats all we get from official centos.